tests/components/androidtv/test_media_player.py

```diff
@@ -299,7 +299,11 @@ def _setup(config):
 async def _test_sources(hass, config0):
     """Test that sources (i.e., apps) are handled correctly for Android TV and Fire TV devices."""
     config = config0.copy()
-    config[DOMAIN][CONF_APPS] = {"com.app.test1": "TEST 1"}
+    config[DOMAIN][CONF_APPS] = {
+        "com.app.test1": "TEST 1",
+        "com.app.test3": None,
```

Comment:
The None would have to change to "" if you were to revert the change I commented on above